ui/apps/platform/src/Containers/NetworkGraph/utils/modelUtils.tsx

```diff
@@ -1,6 +1,17 @@
-import { Model, NodeModel, EdgeModel } from '@patternfly/react-topology';
+import { EdgeModel } from '@patternfly/react-topology';
 
 import { NetworkEntityInfo, Node } from 'types/networkFlow.proto';
+import {
+    CIDRBlockData,
+    CustomModel,
+    CustomNodeData,
+    CustomNodeModel,
+    DeploymentData,
+    ExternalEntitiesData,
+    ExternalNodeModel,
+    NamespaceData,
+    NamespaceNodeModel,
+} from '../types/topology.type';
```

Comment:
Question about naming structure:

Because there are `NamespaceNodeModel` and `ExternalNodeModel`, that makes me wonder if `CustomNodeModel` only applies to deployments ?

If that's the case, would it be better to prefix all the "Custom..." things `Deployment` instead, like `DeploymentNodeModel` ?

Reply:
Oh actually we have 5 different types of nodes:
1. `Namespace`
2. `Deployment`
3. `External`
4. `External Entities`
5. `CIDR Block`

I made `CustomNodeModel` as a type that can be one of these 5 

The hierarchy is as follows:
`Namespace` -- `groups` --> `Deployment`
`External` -- `groups` --> `External Entities`
`External` -- `groups` --> `CIDR Block`

So `Namespace` and `External` are the group nodes and the rest are just nodes (possible children)